common/lib/xmodule/xmodule/course_module.py

```diff
@@ -437,152 +439,308 @@ class CourseFields(object):
+                        "action_text": _("View Course Outline"),
                         "action_external": False,
                     },
                     {
-                        "short_description": _("Add an Instructor-Only Section to Your Outline"),
+                        "short_description": _("Provide transcripts"),
                         "long_description": _(
-                            "Some course authors find using a section for unsorted, in-progress work useful. To do "
-                            "this, create a section and set the release date to the distant future."
+                            "Ensure that transcripts are uploaded for all video and audio content."
                         ),
                         "is_checked": False,
                         "action_url": "CourseOutline",
-                        "action_text": _("Edit Course Outline"),
+                        "action_text": _("View Course Outline"),
                         "action_external": False,
                     },
                 ],
             },
             {
-                "short_description": _("Explore edX's Support Tools"),
+                "short_description": _("Other Instructional Material"),
                 "items": [
                     {
-                        "short_description": _("Explore the Studio Help Forum"),
+                        "short_description": _("Verify third-party material is freely available"),
                         "long_description": _(
-                            "Access the Studio Help forum from the menu that appears when you click your user name "
-                            "in the top right corner of Studio."
+                            "Ensure all required third party content is freely available to students (e.g. open source, public"
+                            "domain, cleared/paid by institution, etc.) and integrated into courseware (not linked out)"
                         ),
                         "is_checked": False,
-                        "action_url": "http://help.edge.edx.org/",
-                        "action_text": _("Visit Studio Help"),
-                        "action_external": True,
+                        "action_url": "CourseOutline",
+                        "action_text": _("View Course Outline"),
+                        "action_external": False,
                     },
                     {
-                        "short_description": _("Enroll in edX 101"),
-                        "long_description": _("Register for edX 101, edX's primer for course creation."),
+                        "short_description": _("Verify paid content (if applicable)"),
+                        "long_description": _(
+                            "Verify that any paid material in the course is optional (not required to achieve learning goals)"
+                            "and clearly marked as paid content."
```

Comment:
If your course offers material that requires payment by leaners, verify that the paid material is not required to achieve learning goals and clearly marked as paid content.